js_modules/dagster-ui/packages/ui-core/src/asset-graph/sidebar/Sidebar.tsx

```diff
@@ -347,7 +349,35 @@ export const AssetGraphExplorerSidebar = React.memo(
           />
         </Box>
         <div>
-          <Container ref={containerRef}>
+          <Container
+            ref={containerRef}
+            onKeyDown={(e) => {
+              let nextIndex = 0;
+              if (e.code === 'ArrowDown' || e.code === 'ArrowUp') {
+                nextIndex = indexOfLastSelectedNodeRef.current + (e.code === 'ArrowDown' ? 1 : -1);
+                indexOfLastSelectedNodeRef.current = nextIndex;
+                e.preventDefault();
+                const nextNode = renderedNodes[nextIndex % renderedNodes.length]!;
+                setSelectedNode(nextNode);
+                selectNode(e, nextNode.id);
+              } else if (e.code === 'ArrowLeft' || e.code === 'ArrowRight') {
+                const open = e.code === 'ArrowRight';
+                setOpenNodes((nodes) => {
+                  const node = renderedNodes[indexOfLastSelectedNode];
+                  if (!node) {
+                    return nodes;
+                  }
+                  const openNodes = new Set(nodes);
+                  if (open) {
+                    openNodes.add(nodePathKey(node));
```

Comment:
I'm not if this has to re-retrieve the node rather than using `selectedNode`?

Reply:
I dont follow